tests/integration/Newsletter/Links/LinksTest.php

```diff
@@ -8,6 +8,7 @@
 use MailPoet\Newsletter\Links\Links;
 use MailPoet\Newsletter\Shortcodes\Categories\Link;
 use MailPoet\Router\Router;
+use function MailPoetVendor\Symfony\Component\DependencyInjection\Loader\Configurator\expr;
```

Comment:
Is this used? I can see where.

Reply:
Good catch! This was added by PHP Storm by accident...